crates/router/src/configs/defaults/payment_connector_required_fields.rs

```diff
@@ -12736,3 +12744,163 @@ impl Default for settings::RequiredFields {
+                        "LA".to_string(),
+                        "LB".to_string(),
+                        "LS".to_string(),
+                        "LR".to_string(),
+                        "LY".to_string(),
+                        "LT".to_string(),
+                        "MO".to_string(),
+                        "MK".to_string(),
+                        "MG".to_string(),
+                        "MW".to_string(),
+                        "MY".to_string(),
+                        "MV".to_string(),
+                        "MR".to_string(),
+                        "MU".to_string(),
+                        "MX".to_string(),
+                        "MD".to_string(),
+                        "MN".to_string(),
+                        "MA".to_string(),
+                        "MZ".to_string(),
+                        "MM".to_string(),
+                        "NA".to_string(),
+                        "NZ".to_string(),
+                        "NI".to_string(),
+                        "NG".to_string(),
+                        "KP".to_string(),
+                        "NO".to_string(),
+                        "AR".to_string(),
+                        "PK".to_string(),
+                        "PG".to_string(),
+                        "PY".to_string(),
+                        "PE".to_string(),
+                        "UY".to_string(),
+                        "PH".to_string(),
+                        "PL".to_string(),
+                        "GB".to_string(),
+                        "QA".to_string(),
+                        "OM".to_string(),
+                        "RO".to_string(),
+                        "RU".to_string(),
+                        "RW".to_string(),
+                        "WS".to_string(),
+                        "SG".to_string(),
+                        "ST".to_string(),
+                        "ZA".to_string(),
+                        "KR".to_string(),
+                        "LK".to_string(),
+                        "SH".to_string(),
+                        "SD".to_string(),
+                        "SR".to_string(),
+                        "SZ".to_string(),
+                        "SE".to_string(),
+                        "CH".to_string(),
+                        "SY".to_string(),
+                        "TW".to_string(),
+                        "TJ".to_string(),
+                        "TZ".to_string(),
+                        "TH".to_string(),
+                        "TT".to_string(),
+                        "TN".to_string(),
+                        "TR".to_string(),
+                        "UG".to_string(),
+                        "UA".to_string(),
+                        "US".to_string(),
+                        "UZ".to_string(),
+                        "VU".to_string(),
+                        "VE".to_string(),
+                        "VN".to_string(),
+                        "ZM".to_string(),
+                        "ZW".to_string(),
```

Comment:
Why do we need this?

Reply:
These are the list of available countries for Worldpay